packages/components/src/accordion/tests/jest/Accordion.ssr.test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment node
```

Comment:
What is it used for? Non SSR jest tests doesn't requires this.

Reply:
This is use to simulate a "server side" environement. This prevents the use of document and window